src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,1391 @@
+import edu.csus.ecs.pc2.list.SubmissionSampleLocation;
+import edu.csus.ecs.pc2.list.SubmissionSolutionList;
+import edu.csus.ecs.pc2.ui.EditFilterPane.ListNames;
+import edu.csus.ecs.pc2.ui.cellRenderer.LinkCellRenderer;
+import edu.csus.ecs.pc2.ui.team.QuickSubmitter;
+
+/**
+ * A UI that to submit files found in a CDP.
+ *
+ *
+ * @author Douglas A. Lane, PC^2 Team, pc2@ecs.csus.edu
+ */
+public class SubmitSampleRunsPane extends JPanePlugin {
+
+    private static final int VERT_PAD = 2;
+    private static final int HORZ_PAD = 20;
+    private static final int SUBMISSION_WAIT_TIMEOUT_MS = 10000;
+
+    private static final int ELAPSED_TIME_COLUMN = 5;
+
+	/**
+	 * ClientSettings key for CDP Path
+	 */
+	private static final String CUSTOM_SUBMIT_SAMPLE_CDP_PATH = "CustomSubmitSampleCDPPath";
+
+	private static final long serialVersionUID = -8862440024499524533L;
+
+	private JTextField cdpTextField;
+
+	private JLabel messageLabel;
+
+	private QuickSubmitter submitter = new QuickSubmitter();
+
+	/**
+	 * List of selected solutions names and dirs.
+	 */
+	private SubmissionSolutionList submissionSolutionList = null;
+
+	private JPanel centerPane = null;
+    private JButton filterButton = null;
+
+    private JScrollPane scrollPane = null;
+    private JTableCustomized runTable = null;
+    private DefaultTableModel runTableModel = null;
+
+    private JPanel messagePanel = null;
+    private JLabel rowCountLabel = null;
+
+    private JCheckBox overrideStopOnError = null;
+
+    /**
+     * User filter
+     */
+    private Filter filter = new Filter();
+
+    private EditFilterFrame editFilterFrame = null;
+
+    private String filterFrameTitle = "Judges' Submissions filter";
+
+    private SampleResultsFrame sampleResultsFrame = null;
+
+    private Log log;
+
+    private List<File> submissionFileList = null;
+    private int currentSubmission = -1;
+    private List<SubmissionSample> submissionList = null;
+    private HashSet<Integer> runsAdded = new HashSet<Integer>();
+
+    private Timer submissionWaitTimer = null;
+    private TimerTask submissionTimerTask = null;
+
+    // Lightish green for match
+    private Color matchColorSuccess = new Color(128, 255, 128);
+    // Lightish red for hard non-match
+    private Color matchColorFailure = new Color(255, 128, 128);
+    // Lightish yellow for take a look/maybe's/indeterminates
+    private Color matchColorMaybe = new Color(255, 255, 128);
+    // Lightish cyan for pending
+    private Color matchColorPending = new Color(128, 255, 255);
+
+
+    // TODO 232 remove debug22Flag
+    private boolean debug22Flag = false;
+
+    // TODO On Admin update of Languages or Update of Problems - clear selected index arrays
+
+	public SubmitSampleRunsPane() {
+		super();
+		setLayout(new BorderLayout());
+
+        add(getMessagePanel(), BorderLayout.NORTH);
+        add(getCenterPanel(), BorderLayout.CENTER);
+        add(getBottomPanel(), BorderLayout.SOUTH);
+
+	}
```

Comment:
indented wrong

Reply:
Fixed and pushed. There were residual tabs from the original code.